backend/prisma/schema.prisma

```diff
@@ -102,6 +106,53 @@ enum InviteStatus {
   NOT_APPLICABLE
 }
 
+model Contact {
+  id                 String                    @id @default(uuid())
+  email              String?
+  firstName          String
+  lastName           String?
+  phoneNumber        String?
+  notes              String?
+  source             String?
+  ownerID            String
+  owner              User                      @relation(fields: [ownerID], references: [id])
+  importantDateEvent ImportantDateEvent[]
+  relationships      UserContactRelationship[]
+  Plan               Plan?                     @relation(fields: [planId], references: [id])
```

Comment:
the fact taht this field is of type `Plan?` instead of `Plan[]?` implies to me that a contact can only be a participant in a max of one Plan. is this the constraint we want to apply?

Reply:
No it is not but I think Hyun might've already changed this in his code.

Reply:
Yeah that makes sense. Fixed!